docs/installation.rst

```diff
@@ -0,0 +1,51 @@
+.. highlight:: shell
+
+============
```

Comment:
Realistically this is a lot of writing (e.g. overhead) for the main point that "microcosm is available on pypi".